src/xcode/ENA/ENA/Source/Services/HealthCertificate/__tests__/Extensions/HealthCertifcateService+Inits.swift

```diff
@@ -0,0 +1,31 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest	
+import Foundation
+import HealthCertificateToolkit
+@testable import ENA
+
+extension HealthCertificateService {
+	convenience init(
+		store: HealthCertificateStoring,
+		dccSignatureVerifier: DCCSignatureVerifying,
+		dscListProvider: DSCListProviding,
+		appConfiguration: AppConfigurationProviding,
+		digitalCovidCertificateAccess: DigitalCovidCertificateAccessProtocol = DigitalCovidCertificateAccess(),
+		notificationCenter: UserNotificationCenter = UNUserNotificationCenter.current(),
```

Comment:
I think we need to pass on these arguments, currently they are ignored

Reply:
Good point, thanks!